### ISSUE TITLE
Including [Elite] into the questname 

### ISSUE THREAD

**Issue by LasterOfDesaster** [AeroScripts/QuestieDev#1281]
## Description

Looking on the map and seeing a quest where I know if it is a elite one only AFTER accepting it and looking in the quest log is kinda annoying. It would be greate if quest had this function as well, since `GetQuestLogTitle(QuestLogID)` returns as second value a tag if the quest is Elite, Dungeon, Group, ...

**Comment by Painstormx** [comment]
I strong support this for map tooltips and quest tracker, it could be like this:
[38] = Normal quest
[38+] = Elite quest
[38D] = Dungeon quest

**Comment by BreakBB** [comment]
Duplicate of #1055
